### PR TITLE
Flex box expose

### PR DESCRIPTION
**Pull request by andreapiras**
This PR exposes Flex and Box so they can be used by consumers. It also enhance the type check for Flex children and improve the testing passing an actual Box to Flex

### CODE REVIEW

File: src/flex/Flex.js [hunk 1]
```diff
@@ -31,17 +31,17 @@ const Flex = props => {
     },
   };
 
-  return (
+  return children || children.length ? (
```
Comment (dozortsev) [inline src/flex/Flex.js]: this check will pass if `children` is an empty array `!![]` equals `true`

File: src/flex/Flex.js [hunk 2]
```diff
@@ -31,17 +31,17 @@ const Flex = props => {
...
     <div className="d-flex" css={style}>
       {React.Children.map(children, child => {
-        if (child) {
+        if (child && child.type && child.type.name === 'Box') {
```
Comment (dozortsev) [inline src/flex/Flex.js]: so this component should accept only `Box` as children?

Comment (andreapiras) [reply src/flex/Flex.js]: Nope you can pass other children but if you pass box is going to pass down the extrapros

File: src/flex/Flex.js (hunk 1 above)
Comment (andreapiras) [reply src/flex/Flex.js]: Actually this check is redundant

**Pull request merged**